nibabel/freesurfer/tests/test_mghformat.py

```diff
@@ -218,14 +217,14 @@ def test_header_updating():
     assert_almost_equal(mgz.affine, exp_aff, 6)
     assert_almost_equal(hdr.get_affine(), exp_aff, 6)
     # Test that initial wonky header elements have not changed
-    assert_equal(hdr['delta'], 1)
+    assert np.array_equal(hdr['delta'],  1)
```

Comment:
This is going to be an array of three ones.

```suggestion
    assert np.all(hdr['delta'] == 1)
```